.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,143 @@
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: Setup node version
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+        cache: 'npm'
+        cache-dependency-path: '**/package-lock.json'
+
+    - name: Setup NPM version
+      run: npm install -g npm@7
+
+    - name: Get NPM cache directory
+      id: npm-cache-dir
+      run: echo "::set-output name=dir::$(npm config get cache)"
+
+    - name: Restore NPM cache
+      id: npm-cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.npm-cache-dir.outputs.dir }}
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+
+    - name: Install node modules
+      run: npm ci --ignore-scripts
+
+    - name: Download prebuild PR artifacts
+      uses: dawidd6/action-download-artifact@d0f291cf39bd21965ea9c4c6e210fc355c3844ed
+      with:
+        workflow: pr-realm-js.yml
+        commit: ${{ github.sha }}
+        path: ${{ github.workspace }}
+        workflow_conclusion: completed
+        github_token: ${{ secrets.REALM_CI_PAT }}
+        name: realm-js-prebuilds
+
+    - name: Read version
+      id: get-version
+      run: |
+        pkgVersion=$(jq -r .version package.json)
+        echo "::set-output name=version::$pkgVersion"
+        prerelease=false
+        if [ '${{ inputs.tag }}' != latest ]; then prerelease=true; fi
+        echo "::set-output name=prerelease::$prerelease"
+      shell: bash
+
+    - name: Npm Publish realm-js-${{ steps.get-version.outputs.version }}
+      run: npm publish --tag '${{ inputs.tag}}' realm-js@${{ steps.get-version.outputs.version }}
+      env:
+        NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+    - name: Find Release PR
+      uses: juliangruber/find-pull-request-action@f9f7484f8237cf8485e5ab826e542ba5dd9e9c6e
+      id: find-pull-request
+      with:
+        branch: ${{ github.ref }}
+
+    - name: Merge Pull Request
+      uses: juliangruber/merge-pull-request-action@8a13f2645ad8b6ada32f829b2fae9c0955a5265d
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        number: ${{ steps.find-pull-request.outputs.number }}
+        method: squash
+
+    - name: Extract Changelog
+      run: scripts/extract-changelog.sh ${{ steps.get-version.outputs.version }} > ExtractedChangelog.md
+
+    - name: Create Github Release
+      uses: ncipollo/release-action@10c84d509b28aae3903113151bfd832314964f2e
+      with:
+        bodyFile: ExtractedChangelog.md
+        name: ${{ steps.get-version.outputs.version }}
+        tag: v${{ steps.get-version.outputs.version }}
+        token: ${{ secrets.GITHUB_TOKEN }}
+        draft: false
+        prerelease: ${{ steps.get-version.outputs.prerelease }}
+
+    - name: Prepare Android and iOS artifacts for uploading
```

Comment:
Not sure if this step is necessary.  The artifacts are published in the npm release

Reply:
And in any case, you would not want to pull the contents of the .xcframework out of that folder.  That should all stay together

Reply:
> Not sure if this step is necessary. The artifacts are published in the npm release

People keep telling me that, but I don't see them :) Which files exactly do you want in the Release page on github? Without this step it's these four:

realm-v10.20.0-napi-v5-darwin-x64.tar.gz
realm-v10.20.0-napi-v5-linux-x64.tar.gz
realm-v10.20.0-napi-v5-win32-ia32.tar.gz
realm-v10.20.0-napi-v5-win32-x64.tar.gz

Reply:
Since the kneth-pr-workflow has been publishing releases, you can find an example here https://github.com/realm/realm-js/releases/tag/v10.20.0

Reply:
> People keep telling me that, but I don't see them :) 

If you make an empty node project and type `npm install realm`, the release binaries will be located in:
* `node_modules/realm/react-native/ios/realm-js-ios.xcframework/`
* `node_modules/realm/react-native/android/src/main/jniLibs/`
You can also verify this by doing `npm pack`, which will output a `.tgz.`.  This should also contain those files.

> Which files exactly do you want in the Release page on github?

I would probably just zip up the entire `react-native` directory.  But i'm not really sure what value any of that provides.  Our users are typically not downloading the artifacts from github.  They are using `npm install`.

Reply:
Okay :) Would it be better to have no release artifacts on the github Release page? That would mean we can cut a couple of steps from the workflow.

The source archive is always there by default.

Reply:
Yes, you can cut those steps out.